src/Common/src/Interop/Unix/libcrypto/Interop.X509.cs

```diff
@@ -29,6 +29,9 @@ internal static partial class libcrypto
         internal static extern SafeX509Handle X509_dup(SafeX509Handle handle);
 
         [DllImport(Libraries.LibCrypto)]
+        internal static extern SafeX509NameHandle X509_NAME_dup(IntPtr handle);
```

Comment:
@bartonjs  I want to understand this , why this needs to be shimmed via pal_x509_name.cpp/h. This I can directly pinvoke..
I assume we did that for the macro functions which we cannot directly pinvoke ?